FaceDetectionServer/REST-API.md

```diff
@@ -0,0 +1,240 @@
+213.73648071289062, 0.6830360293388367], [53.80148696899414, 215.69586181640625,
+0.6367629766464233], [36.029396057128906, 213.71531677246094, 0.6915079951286316]]]}
+```
+
+We'll only show a single pose being recognized, as the amount of data returned for multiple poses can be quite large.
+
+#### Response Details
+Each value in the "poses" array is an x, y coordinate, followed by a confidence value. If more than one body is detected, multiple arrays of coordinates will be included in the response.
+
+The received points represent the start or end of a “bone”, so they must be rendered as pairs. The “0.0” value means that particular segment was not detected.
+
+##### Rendering Implementation for Android
+The following Java arrays are used to render the received coordinates.
+```
+/**
+* The pairs array is a 2D list of the body parts that should be connected together.
+* E.g., 1 for "Neck", 2 for "RShoulder", etc.
+* See https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/output.md
+*/
+private int[][] pairs = {
+       {1,8},{1,2},{1,5},{2,3},{3,4},{5,6},{6,7},{8,9},{9,10},{10,11},{8,12},
+       {12,13},{13,14},{1,0},{0,15},{15,17},{0,16},{16,18},{14,19},
+       {19,20},{14,21},{11,22},{22,23},{11,24}
+};
+
+/**
+* Colors array corresponding to the "pairs" array above. For example, the first pair of
+* coordinates {1,8} will be drawn with the first color in this array, "#ff0055".
+*/
+private String[] colors = {
+       "#ff0055", "#ff0000", "#ff5500", "#ffaa00", "#ffff00", "#aaff00", "#55ff00", "#00ff00",
+       "#ff0000", "#00ff55", "#00ffaa", "#00ffff", "#00aaff", "#0055ff", "#0000ff", "#ff00aa",
+       "#aa00ff", "#ff00ff", "#5500ff", "#0000ff", "#0000ff", "#0000ff", "#00ffff", "#00ffff",
+       "#00ffff"};
+```
+
+#### Example rendered output:
+![Pose Detection Rendered Output](images/render_pose_detection.jpg)
+
+
+## Object Detection
+`/object/detect/`
+
+Used to send an image to the server and get back a set of coordinates and names for any detected objects. Unlike Pose Detection, which requires a GPU, object detection can be ran in CPU-only mode, though it will be considerably slower.
+
+### POST
+`http://<hostname>:8008/object/detect/`
+
+#### Parameters
+| Parameter | Description |
+| - | - |
+| image | The object(s) image. |
+
+#### Return
+JSON string with the following attributes.
+
+| Attribute | Description |
+| - | - |
+| success | "True" if object(s) detected, otherwise "false". |
+| server_processing_time | The time in milliseconds that it took the server to process this image. |
+| gpu_support | Whether the server is using a GPU to accelerate image processing. |
+| objects | A JSON array which may contain a nested JSON array with the following attributes: |
+|  | `rect`: Coordinate of the object |
+|  | `class`: Class name of the object, e.g. "ball", "dog", etc. |
+|  | `confidence`: Confidence level that the object was identified correctly. Range is 0 to 1. |
```

Comment:
Which value represents high and low confidence? Is 0 high confidence like above?